src/app/components/chatbot/chatbot.component.ts

```diff
@@ -0,0 +1,192 @@
+import {
+  AfterViewInit,
+  Component,
+  ElementRef,
+  OnInit,
+  Renderer2,
+  inject,
+} from '@angular/core';
+import { environment } from '../../../environments/environment';
+import { ReCaptchaV3Service } from 'ngx-captcha';
+import { CommonModule } from '@angular/common';
+import { NavigationService } from '../../observables/navigation.service';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'app-mushroomsoft-chatbot',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './chatbot.component.html',
+  styleUrl: './chatbot.component.scss',
+})
+export class ChatbotComponent implements OnInit, AfterViewInit {
+  private readonly copilotUrlToken = environment.COPILOT_URL_TOKEN;
+  public readonly captchaSiteKey = environment.CAPTCHA_SITE_KEY;
+
+  private recaptchaV3Service = inject(ReCaptchaV3Service);
+  private subscription!: Subscription;
+
+  chatOpen = false;
+  showTooltip = true;
+  webchatInitialized = false;
+  iconTransitioning = false;
+
+  openIcon = 'https://cdn-icons-png.flaticon.com/512/6469/6469080.png';
+  closedIcon = 'https://cdn-icons-png.flaticon.com/512/1998/1998597.png';
+
+  tooltipText = 'Hey! Ask me something!';
+
+  constructor(
+    private el: ElementRef,
+    private renderer: Renderer2,
+    private navigationService: NavigationService
+  ) {}
+
+  ngOnInit(): void {
+    const script = this.renderer.createElement('script');
+    script.src =
+      'https://cdn.botframework.com/botframework-webchat/latest/webchat.js';
+    script.crossOrigin = 'anonymous';
+    script.onload = () => console.log('WebChat script loaded.');
+    this.renderer.appendChild(document.body, script);
+
+    this.subscription = this.navigationService.languageObservable.subscribe(
+      (language) => {
+        if (language === 'en') {
+          this.tooltipText = 'Hey! Ask me something!';
+        } else if (language === 'es') {
+          this.tooltipText = '¡Hey! ¡Pregúntame algo!';
```

Comment:
podriamos crear unos archivos jSON para ambos idiomas y dependiendo del idioma muestre el mensaje. Estos tipos de mensajes va a crecer. Ejemplo: https://github.com/mushroomsoft-it/mushroomsoft-it/blob/master/src/app/config/sections.json